lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py

```diff
@@ -59,6 +59,7 @@
     ports:
         description:
             - List of ports exposed within the container group.
```

Comment:
Add a new line:
"- This option is deprecated, using I(ports) under I(containers)"